### PR TITLE
 tooltip and placeholder for search bar in InstructorStudentList need to be changed #2353 

### LINKED ISSUE #2353: tooltip and placeholder for search bar in InstructorStudentList need to be changed
Currently the tooltip says "student's name" although the search bar searches all the values in the table. The placeholder can also be changed to reflect this better.

### PR DESCRIPTION
**Pull request by ngzhian**
Updated the HTML in test cases as well
fixes #2353

### CODE REVIEW

**Comment by junhaoyap** [comment]
I'll take a look and ensure the test all passes over Hari Raya and let you know :)

**Comment by junhaoyap** [comment]
Looks good, reflects what is searched better.

File: src/main/webapp/WEB-INF/tags/instructor/home/search.tag [hunk 1]
```diff
@@ -13,7 +13,7 @@
 			               class="form-control"
 			               data-toggle="tooltip"
 			               data-placement="top"
-			               placeholder="Student Name">
+			               placeholder="e.g. Charles Shultz, charles@gmail.tmt">
```
Comment (damithc) [inline src/main/webapp/WEB-INF/tags/instructor/home/search.tag]: `.tmt` is what we used to identify test accounts. `.com`?

Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/instructor/home/search.tag]: Ah right, that, I saw gmail and assumed it was .com, @ngzhian could you change that? :)

**Comment by ngzhian** [comment]
@junhaoyap ready for review

**Comment by junhaoyap** [comment]
@ngzhian Thanks, looks good, I'll merge it in later :)

**Pull request merged**